web_src/js/publicPath.js

```diff
@@ -0,0 +1,16 @@
+/* This sets up webpack's chunk loading to load resources from the same
+  directory where it loaded index.js from. This file must be imported
+  before any lazy-loading is being attempted. */
+
+if (document.currentScript && document.currentScript.src) {
+  const url = new URL(document.currentScript.src);
+  __webpack_public_path__ = `${url.pathname.replace(/\/[^/]*$/, '')}/`;
+} else {
+  // compat: IE11
+  const script = document.querySelector('script[src*="/index.js"]');
+  if (script) {
+    __webpack_public_path__ = `${script.getAttribute('src').replace(/\/[^/]*$/, '')}/`;
+  } else {
+    __webpack_public_path__ = '/js/';
```

Comment:
Shouldn't this be `/public/js/`?

Reply:
`/js/` should be correct, the final script URL uses that path:

```
http://localhost:3000/js/index.js?v=e96761133e24f3d40ae17fc5a9e38ef7
```

Reply:
It depends on if user have added a subpath. So this may break some deployment with subpath.

Reply:
This is the 3rd fallback which I think is pretty much impossible to hit in any browser so I think it's fine to not support subpath in that fallback.

Reply:
Given that the case is probably never hit, I'm going to remove it altogether.